packages/cli/test/utils/crucible/assertions/blobsAssertion.ts

```diff
@@ -28,7 +29,8 @@ export function createBlobsAssertion(
       // slot starts from 1, so we add up 1
       if (slot <= numberOfBlobs + 1 && node.id === "node-1") {
         const {blobs, kzgCommitments, blobVersionedHashes, kzgProofs} = generateBlobsForTransaction(1);
-        const nonce = await node.execution.provider?.eth.getTransactionCount(EL_GENESIS_ACCOUNT);
+        // since the delay_fcu flag, calling to get nonce here will cause duplicate
+        // const nonce = await node.execution.provider?.eth.getTransactionCount(EL_GENESIS_ACCOUNT);
```

Comment:
this seems very strange, we just query data from EL, what's the exact issue here? it's not really clear from the comment

Reply:
It looks like we didn't even need to explicitly define nonce

https://github.com/web3/web3.js/blob/bf1691765bd9d4b0f7a4479e915207707d69226d/packages/web3-eth/src/utils/transaction_builder.ts#L149

If nonce is missing, web3 will automatically calls `getTransactionCount` unless we want to override the nonce value which is the case here